win32ss/user/ntuser/winpos.c

```diff
@@ -3103,6 +3105,83 @@ IntDeferWindowPos( HDWP hdwp,
     return retvalue;
 }
 
+/* Win: xxxImeWindowPosChanged */
```

Comment:
Win: comments strongly discouraged (cf. the mattermost discussion that happened some months ago).

Reply:
Deleted.